app/setUpJestServer.js

```diff
@@ -1,3 +1,4 @@
+process.env.EXTERNAL_SERVICES = true;
```

Comment:
I guess the goal is to always have this true for tests. Just out of curiosity: is this a common pattern in JS (i.e. setting environment variables in code)? 
Also, can this go after the import? My eslint complains (with just a warning).

Reply:
I think it's fine for the purposes of the test. I moved the line.